### PR TITLE
Changes to QA related files (e.g., `method.md`)

### PR DESCRIPTION
**Pull request by sergio-mena**
Contributes to #31

Cumulative changes to QA-related markdown files that were performed to improve the text as we ran pre-release QA tests for `v0.38.x`.

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

File: docs/qa/method.md [hunk 1]
```diff
@@ -62,19 +62,35 @@ This section explains how the tests were carried out for reproducibility purpose
      desired proportion of nodes running each of the two configured versions.
 4. Follow steps 5-10 of the `README.md` to configure and start the 200 node testnet
     * WARNING: Do NOT forget to run `make terraform-destroy` as soon as you are done with the tests (see step 9)
-5. As a sanity check, connect to the Prometheus node's web interface and check the graph for the `COMETBFT_CONSENSUS_HEIGHT` metric.
-   All nodes should be increasing their heights.
-6. You now need to start the load runner that will produce transaction load
+5. As a sanity check, connect to the Prometheus node's web interface (port 9090)
```
Comment (lasarojc) [inline docs/qa/method.md:65]: Prometheus is listed 2 in the requirements above (line 44).
Do they mean different things?

File: docs/qa/method.md [hunk 2]
```diff
@@ -39,9 +39,10 @@ _200 Node Test_, and _Rotating Nodes Test_.
 ### Requirements for Result Extraction
 
 * Matlab or Octave
```
Comment (sergio-mena) [inline docs/qa/method.md:41]:
```suggestion
```

File: docs/qa/method.md [hunk 3]
```diff
@@ -229,29 +254,32 @@ This section explains how the tests were carried out for reproducibility purpose
...
       [this paragraph](CometBFT-QA-34.md#finding-the-saturation-point) for further info)
+    * `Z` (in seconds) should be big enough to keep running throughout the test, until we manually stop it in step 9.
+      In principle, a good value for `Z` is `7200` (2 hours)
 8. Run `make rotate` to start the script that creates the ephemeral nodes, and kills them when they are caught up.
-    * WARNING: If you run this command from your laptop, the laptop needs to be up and connected for full length
+    * WARNING: If you run this command from your laptop, the laptop needs to be up and connected for the full length
       of the experiment.
-9. When the height of the chain reaches 3000, stop the `make rotate` script
+    * [This](http://161.35.205.129:9090/classic/graph?g0.range_input=100m&g0.expr=cometbft_consensus_height%7Bjob%3D~%22ephemeral.*%22%7D%20or%20cometbft_blocksync_latest_block_height%7Bjob%3D~%22ephemeral.*%22%7D&g0.tab=0&g1.range_input=100m&g1.expr=cometbft_mempool_size%7Bjob!~%22ephemeral.*%22%7D&g1.tab=0&g2.range_input=100m&g2.expr=cometbft_consensus_num_txs%7Bjob!~%22ephemeral.*%22%7D&g2.tab=0)
```
Comment (sergio-mena) [inline docs/qa/method.md:265]:
```suggestion
    * [This](http://<PROMETHEUS-NODE-IP>:9090/classic/graph?g0.range_input=100m&g0.expr=cometbft_consensus_height%7Bjob%3D~%22ephemeral.*%22%7D%20or%20cometbft_blocksync_latest_block_height%7Bjob%3D~%22ephemeral.*%22%7D&g0.tab=0&g1.range_input=100m&g1.expr=cometbft_mempool_size%7Bjob!~%22ephemeral.*%22%7D&g1.tab=0&g2.range_input=100m&g2.expr=cometbft_consensus_num_txs%7Bjob!~%22ephemeral.*%22%7D&g2.tab=0)
```

**Review by lasarojc — APPROVED** [review]

**Review by sergio-mena — COMMENTED** [review]
@lasarojc's and @hvanz's changes LGTM. Approve.

**Pull request merged**